### ISSUE TITLE
Include Shellcheck utility

### ISSUE THREAD

**Issue by otaviobp** [clearlinux/distribution#271]
Shellcheck is a static analysis tool for shellscripts and it's available at: https://github.com/koalaman/shellcheck

As it uses cabal build system and this isn't suported by autospec, I haven't built this myself.

**Comment by otaviobp** [comment]
> Seems reasonable, suggested bundle? can you make an initial package?

It looks like something easy to do as long as we had cabal on clear and cabal support on autospec. Adding support for cabal on the other hand, I'm not sure how easy it will be and I haven't tried that yet. It's in my future list, but I am not going to work on that now. Just sending the idea, maybe there's someone else interested on checking that

**Comment by mdhorn** [comment]
If this isn't difficult to package, this would be helpful for the validation of several of our shell scripts.
FYI @karthikprabhuvinod

**Comment by karthikprabhuvinod** [comment]
@otaviobp @mdhorn : I read in Rob's email that this has already been done! Is it same linter?

**Comment by mdhorn** [comment]
> @otaviobp @mdhorn : I read in Rob's email that this has already been done! Is it same linter?

No, I think Rob misinterpreted my update that we NEED this still to add as a standard linter.
This request is not done as the checker in not in our builds.

**Comment by damageboy** [comment]
Would be nice to include this in clearlinux

**Comment by namm2** [comment]
Can we expect Shellcheck to be available in 2021?
